data/constants.py

```diff
@@ -3,6 +3,12 @@
 myId = "171671190631481345"
 didierId = "680510935164911730"
 coolerDidierId = "728361496874057812"
+
+didier_identities = ["680510935164911730",  # Didier
+                     "728361496874057812",  # coolerDidier
+                     "831899971686760558",  # Communist Didier
+                     ]
+
```

Comment:
You can use `client.user.id` to get the (current) bot's ID without hardcoding it in